tests/test_bgp_af.rb

```diff
@@ -106,6 +118,38 @@ def test_collection_not_empty
     end
   end
 
+  def config_ios_xr_dependencies(asn)
+    # These dependencies are required on ios xr
```

Comment:
@RichWellum @gvlaemyn What was the conclusion here? Is this going to be covered by Rich's open user story?